pkg/process/config/config.go

```diff
@@ -305,6 +307,11 @@ func NewAgentConfig(loggerName config.LoggerName, yamlPath, netYamlPath string)
 		cfg.Windows.ArgsRefreshInterval = -1
 	}
 
+	// activate the pod collection if we have the cluster name set
+	if canAccessContainers && cfg.KubeClusterName != "" {
```

Comment:
I think for now we should also have an explicit `cfg.enablePodCollection` - thoughts?

Reply:
I agree, especially we are collecting yaml content which could be big for pod collection, customers might/might not want that. But now we have more flags to manage so that's also not so good......

Reply:
The idea as we want to have the `cluster_name` mandatory to enable the collection was to use it as the enablement parameter as well, to be reused in the cluster-agent as in it's own section: `k8s_explorer.cluster_name` (I could add `k8s_explorer.enabled` this feels redundant but might be more explicit?)

Reply:
Yeah I think we should have an explicit enabled flag for now - we can always remove it later if we want to change the onboarding experience